runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewDelegate.java

```diff
@@ -96,8 +96,10 @@ public static void init(Context libContext, Context appContext) {
 
         try {
             if (libContext == null) {
-                init(appContext);
+                Log.d(TAG, "Initializing plain context");
+                init(libContext);
```

Comment:
typo? it shall be appContext here. Also, please remove the debug messages and keep the change as clean as possible. Thanks.